src/extensions/surveys.tsx

```diff
@@ -88,6 +85,7 @@ export const callSurveys = (posthog: PostHog, forceReload: boolean = false) => {
                 }
 
                 if (!localStorage.getItem(`seenSurvey_${survey.id}`)) {
+                    localStorage.setItem(`seenSurvey_${survey.id}`, 'true')
```

Comment:
hmm, this can't be here imo, because it means if the user 'forgot' to interact with the survey and refreshed the page, we wouldn't show it again. Instead, we should keep showing it until they press dismiss or submit.

Reply:
fixed now